clientv3/integration/kv_test.go

```diff
@@ -30,8 +30,35 @@ import (
 	"github.com/coreos/etcd/pkg/testutil"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
 )
 
+func TestHealth(t *testing.T) {
```

Comment:
this should be in `integration/` instead of `clientv3/integration/` since it's mostly only testing the server